web/api/netdata-swagger.yaml

```diff
@@ -627,6 +627,27 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/alarms"
+  /alarms:
```

Comment:
alarms_values here?

Reply:
Yes, you are correct. I changed it now, thank you!